test/conformance/testing/include/uur/fixtures.h

```diff
@@ -568,6 +568,36 @@ template <class T> struct urUSMPoolTestWithParam : urContextTestWithParam<T> {
     ur_usm_pool_handle_t pool;
 };
 
+struct urPhysicalMemTest : urContextTest {
+
```

Comment:
I don't like the newline after declaration thing and it isn't 100% followed in the file, can we chuck it?

Reply:
👍